server/server.go

```diff
@@ -159,7 +159,8 @@ func (server *Server) getDatabaseConfig() (string, string, bool, bool) {
 	}
 	databaseDropOnCreate := config.GetBool("database/drop_on_create", false)
 	databaseCascade := config.GetBool("database/cascade_delete", false)
-	return databaseType, databaseConnection, databaseDropOnCreate, databaseCascade
+	databaseAutoMigrate := config.GetBool("database/auto_migrate", true)
```

Comment:
@mmatczuk , It seems no_init flag also disables migration, which is exactly what we need. Perhaps this change is not required, then. @przemyslaw-dobrowolski-cl ?